app/Services/DislikeService.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Services;
+
+use App\Dislike;
+use App\Like;
+use App\Movie;
+use App\User;
+use Illuminate\Http\JsonResponse;
+
+class DislikeService
+{
+    /**
+     * Creates new like for a movie in Likes table.
+     *
+     * @param User $user
+     * @param User $movie
+     * @return Dislike
+     */
+    public function create(User $user, Movie $movie): Dislike
+    {
+        $dislike['movie_id'] = $movie->id;
+        $dislike['user_id'] = $user->id;
+
+        return Dislike::create($dislike);
+    }
+
+    /**
+     * Deletes dislikelike for a movie in Dislikes table.
+     *
+     * @param User $user
+     * @param User $movie
+     * @return Response
+     */
+    public function destroy(User $user, Movie $movie)
+    {
+        Dislike::where([['movie_id', '=', $movie->id], ['user_id', '=', $user->id]])->delete();
+        $response = new JsonResponse([
+            'message' => 'Dislike removed successfully',
+        ], 200);
+        return $response;
```

Comment:
mozes samo :

```
return [
            'message' => 'Dislike removed successfully',
        ]
```
Bice sam od sebe json i 200

Reply:
Ali ono i ne treba ti bas poruka kontam, 200 je dovoljan indikator, ja bih radije vratio to sto si kreirao nego samo poruku, tako da bolje vrati resurs

Reply:
A da vidim da nemas resurs, tako da vrati samo 200 i kontam to je to :D